src/utils/string.ts

```diff
@@ -0,0 +1,3 @@
+export function pluralize(count: number, singular: string) {
+  return count === 1 ? singular : `${singular}s`;
+}
```

Comment:
lovely

Reply:
will need to upgrade in the future for weird plurals but works for now lol